zapisy/apps/offer/vote/assets/sortable-table.scss

```
@@ -0,0 +1,30 @@
+td:first-child {
+  width: 60%;
+}
+
+th.tablesorter-header:not(.sorter-false) {
+  cursor: pointer;
+
+  &:hover {
+    background-color: rgba(0, 0, 0, 0.075);
+  }
+
+  .tablesorter-header-inner {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    .tablesorter-icon {
+      color: #bfbfc1;
+      margin-left: 0.75rem;
+
+      @media (max-width: 768px) {
```

No, mało to semantyczne. Jeśli dobrze rozumiem, poniższe będzie miało de facto ten sam efekt, a nie hardkoduje wartości zmiennej w dodatkowym miejscu (tak, zapewne mamy ich już sporo, ale każdy moment jest dobry, żeby przestać – czy nie?)
```suggestion
      @include media-breakpoint-down(md) {
```

---

Ano, to jest znacznie lepszy zapis. Musiałem też dodać w tym pliku dodatkowe importy, aby móc tej formy użyć.